src/Settings/components/transliteration.jsx

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from "react";
+import { Text } from "react-native";
+import { ListItem, BottomSheet, Avatar, Icon, Switch } from "@rneui/themed";
+import { useSelector } from "react-redux";
+import PropTypes from "prop-types";
+import STRINGS from "../../common/localization";
+import {
+  setTransliteration,
+  toggleTransliteration,
+  TRANSLITERATION_LANGUAGES,
+} from "../../common/actions";
+import colors from "../../common/colors";
+import styles from "../styles";
+
+const renderItem = (item, dispatch, isNightMode, toggleVisible, transliterationLanguage) => {
+  return (
+    <ListItem
+      key={item.key}
+      bottomDivider
+      containerStyle={[
+        { backgroundColor: isNightMode ? colors.NIGHT_GREY_COLOR : colors.WHITE_COLOR },
+      ]}
+      onPress={() => {
+        toggleVisible(false);
+        dispatch(setTransliteration(item.key));
+      }}
+    >
+      <ListItem.Content>
+        <ListItem.Title style={[isNightMode && { color: colors.WHITE_COLOR }]}>
+          {item.title}
+        </ListItem.Title>
+      </ListItem.Content>
+      {transliterationLanguage === item.key && (
+        <Icon color={isNightMode && colors.WHITE_COLOR} name="check" />
+      )}
+    </ListItem>
+  );
+};
+function TransliterationComponent({ isNightMode, dispatch }) {
+  const [isVisible, toggleVisible] = useState(false);
+  const { transliterationLanguage, isTransliteration } = useSelector((state) => state);
+
+  return (
+    <>
+      <ListItem
+        bottomDivider
+        containerStyle={[
+          { backgroundColor: isNightMode ? colors.NIGHT_GREY_COLOR : colors.WHITE_COLOR },
+        ]}
+      >
+        <Avatar source={require("../../../images/romanizeicon.png")} />
```

Comment:
better to require the image on the top and use it here

Reply:
fixed